flake.nix

```diff
@@ -0,0 +1,35 @@
+{
+  description = "provides the libvfn package for NixOS and Nix environments";
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-24.05";
+
+  };
+
+  outputs = { self, nixpkgs }:
+    let
+      libvfnVersion = "4.0.1";
```

Comment:
@jwdevantier wanted to pick up this PR. How is this supposed to be handled? Will the flake always refer to the latest released version, or can it be used for `main`?